test/schema/gene/gene_families.js

```diff
@@ -0,0 +1,53 @@
+import schema from "../../../schema"
+import { runQuery } from "../../utils"
+
+describe("GeneFamilies", () => {
+  const GeneFamilies = schema.__get__("GeneFamilies")
+  let gravity = null
+  const api_data = [
+    {
+      id: "design-concepts-and-techniques",
+      name: "Design Concepts and Techniques",
+    },
+    {
+      id: "furniture-and-lighting",
+      name: "Furniture & Lighting",
+    },
+  ]
+
+  beforeEach(() => {
+    gravity = sinon.stub()
+    gravity.returns(Promise.resolve(api_data))
+    GeneFamilies.__Rewire__("gravity", gravity)
+  })
+
+  afterEach(() => {
+    GeneFamilies.__ResetDependency__("gravity")
+  })
+
+  it("returns a list of gene families", () => {
+    const query = `
+      {
+        gene_families {
+          id
+          name
+        }
+      }
+    `
+
+    return runQuery(query).then(data => {
```

Comment:
Once you’ve replaced the use of the `gravity` loader with a per-type loader you can get rid of all the rewiring in this test case. Instead you you use DI by passing in a stub function to [the GraphQL query execution](https://github.com/artsy/metaphysics/blob/2d9de40d40a537a17447d7094589f8c9065e17ba/test/utils.js#L17) that acts as your specific loader:

```js
const geneFamiliesLoader = () => Promise.resolve(api_data)
runQuery(query, { geneFamiliesLoader })
```